src/test/py/bazel/bazel_windows_cpp_test.py

```diff
@@ -278,19 +280,26 @@ def testBuildDifferentCcBinariesDependOnConflictDLLs(self):
             '  linkstatic = 0,'
             ')',
         ])
+    bazel_bin = self.getBazelInfo('bazel-bin')
     self.ScratchFile('main/other_main.cc', ['int main() {return 0;}'])
 
     # Building //main:main should succeed
-    exit_code, _, stderr = self.RunBazel(['build', '//main:main'])
+    # We should check the A_{hash}.dll exist.
```

Comment:
Maybe check if there is a file name as a pattern like "A_*.dll"? We don't have to calculate the actual hash.